pyLOM/POD/wrapper.pyx

```diff
@@ -79,11 +81,13 @@ def _srun(float[:,:] X, int remove_mean):
 	cdef int m = X.shape[0], n = X.shape[1], mn = min(m,n), retval
 	cdef float *X_mean
 	cdef float *Y
+	cdef unsigned int seed = 19
```

Comment:
Well here is where you should call the time module...
```
from libc.time   cimport time
cdef unsigned int seed = <int>time(NULL)
```

Reply:
I just wanted the seed to be Lamine Yamal's number :(

Reply:
Please fix